### PR TITLE
Move pack Hey to DevelopersTools

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5042

## Description
Deprecating Hey pack and moving it's content (the script) to Developer tools 

## Minimum version of Cortex XSOAR
- [X] 6.0.0

## Does it break backward compatibility?
   - [X] Yes

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/.pack-ignore [hunk 1]
```diff
@@ -73,6 +73,30 @@ ignore=BA119
 [file:HideFieldsOnNewIncident.yml]
 ignore=SC105
 
+[file:TAXII2_Performance_Test_-_Concurrency.yml]
```
Comment (Shellyber) [inline Packs/CommonScripts/.pack-ignore:76]: Why do we have the change here in CommonScripts? we don't touch this pack in this PR.
Please revert

File: Packs/Hey/pack_metadata.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-    "name": "Hey",
+    "name": "Hey (deprecated)",
     "description": "Use rakyll/hey to test a web application with a load of requests.",
```
Comment (Shellyber) [inline Packs/Hey/pack_metadata.json:3]: Add deprecated to the description as well

File: Packs/DeveloperTools/pack_metadata.json [hunk 3]
```diff
@@ -32,6 +32,10 @@
         "CommonTypes": {
             "mandatory": true,
             "display_name": "Common Types"
+        },
+        "GetServerURL": {
```
Comment (Shellyber) [inline Packs/DeveloperTools/pack_metadata.json:36]: remove this please.
GetServerUrl is moving to CommonScripts so no need to add it here

**Review by Shellyber — COMMENTED** [review]
Nice

File: Packs/Hey/ReleaseNotes/1_0_3.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+### Hey
+##### Hey
+- Note: Moved to the *DeveloperTools* pack.
```
Comment (Shellyber) [inline Packs/Hey/ReleaseNotes/1_0_3.md:3]:
```suggestion
- Deprecated. Use the *DeveloperTools* pack instead.
```

File: Packs/CommonScripts/.pack-ignore (hunk 1 above)
Comment (maimorag) [reply Packs/CommonScripts/.pack-ignore:76]: ok, reverting all changes in common scripts

File: Packs/Hey/pack_metadata.json (hunk 2 above)
Comment (maimorag) [reply Packs/Hey/pack_metadata.json:3]: added

File: Packs/DeveloperTools/pack_metadata.json (hunk 3 above)
Comment (maimorag) [reply Packs/DeveloperTools/pack_metadata.json:36]: done

File: Packs/Hey/ReleaseNotes/1_0_3.md (hunk 4 above)
Comment (maimorag) [reply Packs/Hey/ReleaseNotes/1_0_3.md:3]: done

File: Packs/Hey/pack_metadata.json [hunk 5]
```diff
@@ -1,8 +1,8 @@
 {
-    "name": "Hey",
-    "description": "Use rakyll/hey to test a web application with a load of requests.",
+    "name": "Hey (deprecated)",
+    "description": "Deprecated. Use the script 'Hey' from 'Developer Tools' pack instead",
```
Comment (Shellyber) [inline Packs/Hey/pack_metadata.json:3]:
```suggestion
    "description": "Deprecated. Use the script Hey from Developer Tools pack instead",
```

**Review by Shellyber — APPROVED** [review]
See my last note 
Nice

File: Packs/Hey/pack_metadata.json (hunk 5 above)
Comment (maimorag) [reply Packs/Hey/pack_metadata.json]: done

File: Packs/Hey/ReleaseNotes/1_0_3.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+#### Hey
```
Comment (Shellyber) [inline Packs/Hey/ReleaseNotes/1_0_3.md:1]: Wrong header here as well

Comment (maimorag) [reply Packs/Hey/ReleaseNotes/1_0_3.md]: fixed

**Comment by maimorag** [comment]
the ci-cd fails due to a bug in the graph build:
https://jira-hq.paloaltonetworks.local/browse/CIAC-5194 

there weren't any changes in the failed test playbooks (as @DeanArbel  and I saw last week, the last successful run we viewed was
https://code.pan.run/xsoar/content/-/jobs/20115343  )

File: Packs/DeveloperTools/Scripts/Hey/Hey.yml [hunk 7]
```diff
@@ -1,7 +1,7 @@
 commonfields:
   id: Hey
   version: -1
-fromversion: 6.0.0
+fromversion: 6.5.0
```
Comment (Shellyber) [inline Packs/DeveloperTools/Scripts/Hey/Hey.yml:4]: please revert the fromversion to what it was

**Review by Shellyber — APPROVED** [review]
Nice.
See my comment before merge

**Comment by maimorag** [comment]
build filed on the following validation errors due to moving the script to another pack :
Packs/Hey/ReleaseNotes/1_0_3.md - [RN114]
Packs/Hey/ReleaseNotes/1_0_3.md - [RN113]

@dantavori can you force merge ?

**Pull request merged**